.github/workflows/live_deployment.yml

```diff
@@ -0,0 +1,25 @@
+name: Live Deployment to PyPI
+
+on:
+  workflow_dispatch
+
+jobs:
+
+  live-deployment-to-pypi:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v2
+    - name: Setup template credentials
+      run: cp snowshu/templates/credentials.yml tests/assets/integration/credentials.yml
```

Comment:
I don't think it would get picked up, but we don't need to add the test credentials for publishing pipelines.